src/github.com/getlantern/analytics/analytics.go

```diff
@@ -4,22 +4,22 @@ import (
 	"bytes"
 	"net/http"
 	"net/url"
+	"runtime"
+	"strconv"
 
+	"github.com/getlantern/flashlight/util"
 	"github.com/getlantern/golog"
 )
 
 const (
-	ApiEndpoint          = `https://ssl.google-analytics.com/collect`
-	ProtocolVersion      = "1"
-	DefaultClientVersion = "1"
-	TrackingId           = "UA-21815217-2"
-	DefaultClientId      = "555"
+	ApiEndpoint       = `https://ssl.google-analytics.com/collect`
+	ProtocolVersion   = "1"
+	DefaultInstanceId = "1260961011.1389432370"
 )
 
 var (
-	log = golog.LoggerFor("analytics")
-
-	defaultHttpClient = &http.Client{}
+	log        = golog.LoggerFor("analytics")
+	httpClient = &http.Client{}
```

Comment:
I would just declare the httpClient and not create one here. I realize that's not really a change you made, but without that the later check for httpClient == nil doesn't make any sense since it will never be nil, and initially we'll try to make direct analytics call until util.HttpClient returns.